decentraland/unity-renderer
# fix: ab detection functionality not working in the build

## What does this PR change?

This PR fixes ab detection functionality bug of it working only in the Unity Editor, but not in the web build.

The main bug was on kernel-renderer communication side. It was "Main" bridge used on Renderer side and "Bridges" bridge used on Kernel side, though somehow it worked in the Unity Editor. I changed Renderer to use "Bridges" bridge and it started working in the build. I also made raw GLTF tagging and painting specific, previously it was "paint red everything that is not AB".

This is the original PR for ab detection functionality:

https://github.com/decentraland/unity-renderer/pull/3099

Closes #2911 

...

## How to test the changes?

[https://play.decentraland.zone/?renderer-branch=ab-detection-functionality](https://play.decentraland.zone/?renderer-branch=ab-detection-functionality

Use chat command /detectabs at some place with ABs enabled. The shapes should turn green.

Use chat command /detectabs off . The shapes should turn to their usual look.

Use chat command /detectabs scene . The shapes should turn green only within current scene.

Use chat command /detectabs scene off . The shapes of that last scene should turn to their usual look.

Command /detectabs off should also turn all shapes to normal look after /detectabs scene

Repeat 1-2 with ABs disabled, the color should be red instead of green. To test with asset bundles disabled use this link:
https://play.decentraland.zone/?renderer-branch=ab-detection-functionality&DISABLE_ASSET_BUNDLES=

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
